atomic_reactor/util.py

```diff
@@ -733,7 +733,8 @@ def get_manifest_digests(image, registry, insecure=False, dockercfg_path=None,
                            media_type, response.status_code)
             continue
 
-        response.raise_for_status()
+        if response.status_code is not None:
+            response.raise_for_status()
```

Comment:
I think we should provide a status code in the mock Responses instead of doing this.